poodle/src/components/Mypage/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import * as S from '@/styles/Mypage/ProgressBar'
+
+const dummyData = {
+    '전형구분': 'false',
+    '인적사항': 'false',
+    '성적입력': 'false',
+    '자기소개서': 'true',
+}
+
+const ProgressBar: React.FC<{}> = () => {
+
+    return (
+        <S.ProgressBarWrapper>
+            <S.ProgressBox>
+                {
+                    Object.entries(dummyData).map((data, index) => (
+                        <ProgressItem 
+                            key={index}
```

Comment:
```suggestion
                            key={data[0]}
```
index 값을 key 로 사용하는 건 의미가 없습니다.

Reply:
(덧붙여서) 의미가 없다는말은 기본적으로 index를 사용한다는 뜻입니다 
https://reactjs.org/docs/lists-and-keys.html#keys
> If you choose not to assign an explicit key to list items then React will default to using indexes as keys

Reply:
우와? 그런 사실이 있엇군요.
key를 index말고 고유한 값으로 수정했습니다.